MAUI/DataForm/Editing.md

```diff
@@ -0,0 +1,302 @@
+---
+layout: post
```

Comment:
remove editing file, it was done with Commit UG

Reply:
File removed